components/ws-manager-api/core.proto

```diff
@@ -229,20 +229,23 @@ message WorkspaceSpec {
     // workspace_image is the name of the Docker image this workspace runs
     string workspace_image = 1;
 
+    // ide_image is the name of the Docker image used as IDE
+    string ide_image = 2;
```

Comment:
Wouldn't it be better to not re-use the field labels so we are backwards compatible? I know we discussed and concluded that we might not use old and new version in one cluster but... that would cost us nothing, or am I mistaken?

Reply:
It probably would - it's more vanity than anything else.
The argument here is about the same as for the DB migrations: either it's now or never.